sdk/core/core-util/src/delay.ts

```diff
@@ -1,11 +1,63 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { AbortError, AbortSignalLike } from "@azure/abort-controller";
+import { isDefined } from "./typeGuards";
+
+const StandardAbortMessage = "The operation was aborted.";
+
+/**
+ * Options for support abort functionality for the delay method
+ */
+export interface DelayOptions {
+  abortSignal?: AbortSignalLike;
+  abortErrorMsg?: string;
```

Comment:
can you add docs for the two properties?

Reply:
Done!